docker/images/sample-tcstore-client/Dockerfile

```diff
@@ -26,7 +31,7 @@ WORKDIR $SAG_HOME
 # make the JVM aware of the container memory constraints, since JDK 8u131
 # see https://blog.csanchez.org/2017/05/31/running-a-jvm-in-a-container-without-getting-killed
 ENV JAVA_OPTS "-XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap"
-ENV CLASSPATH ".:/opt/softwareag/client/store/*:/opt/softwareag/client/ehcache/*:/opt/softwareag/client/lib/*:/opt/softwareag/client/logging/*:/opt/softwareag/client/logging/impl/*"
+ENV CLASSPATH ".:/opt/softwareag/client/ehcache/*:/opt/softwareag/client/lib/*:/opt/softwareag/client/logging/*:/opt/softwareag/client/logging/impl/*"
```

Comment:
Currently this puts both ehcache and store classes on the class path... won't break anything, no idea if anyone even cares.

Reply:
We decided not to fix this